ledger/accountdb.go

```diff
@@ -240,15 +297,40 @@ type compactAccountDeltas struct {
 	misses []int
 }
 
+// onlineAccountDelta track all changes of account state within a range
+// used in conjunction wih compactOnlineAccountDeltas to group and represent per-account changes
+type onlineAccountDelta struct {
+	oldAcct           persistedOnlineAccountData
```

Comment:
I got confused. The reason oldAcct is a single account, and newAcct is a slice, is because oldAcct is the latest update for that account, correct? and newAcct is a series of account updates to flush given a commit range? maybe a comment here to indicate what oldAcct is used for? (it is basically compared to newAcct[0] to decide if this is a new entry or not?)

Reply:
this is the correct understanding. I'll update the comment